### ISSUE TITLE
Windows Registry not modified with 0.4.0 update

### ISSUE THREAD

**Issue by Interagierende-Systeme** [zealdocs/zeal#782]
The DefaultIcon and Command variables in the Windows Registry are not modified with the 0.4.0 update.
The path is still "Program Files(x86)" instead of "Program Files" where Zeal is now installed by default for me, and must be manually changed for Zeal to work through URL calls.

Is this for other users as well?

**Comment by trollixx** [comment]
1. Was the previous version removed at all?
2. Is it possible that you installed protocol handlers via `zeal --register`?

**Comment by Interagierende-Systeme** [comment]
1. Yes, the previous version is removed from Program Files (x86), and the new version is installed at "Program Files/Zeal".
2. No, I only installed through the setup file.

I am not facing this problem on Windows 10, so I believe it is a problem with my Windows Virtual Machine (Win 7). 

I tested again by un-installing Zeal 0.4.0, removing manually the registry entries dash, and dash-plugin and then installing the older Zeal. Now, I uninstalled the older Zeal and installed Zeal 0.4.0 and this time the registry values are updated. However, when I do this without at some point manually removing the registry entries, they do not seem to get updated. Seems quite weird, but perhaps it is only a virtual machine or Windows 7 problem.
